example/lib/attribution.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+import 'package:maplibre_gl/mapbox_gl.dart';
+
+import 'page.dart';
+
+class AttributionPage extends ExamplePage {
+  const AttributionPage({super.key})
+      : super(const Icon(Icons.thumb_up), 'Attribution');
+
+  @override
+  Widget build(BuildContext context) {
+    return const AttributionBody();
+  }
+}
+
+class AttributionBody extends StatefulWidget {
+  const AttributionBody({super.key});
+
+  @override
+  State<AttributionBody> createState() => _AttributionBodyState();
+}
+
+class _AttributionBodyState extends State<AttributionBody> {
+  AttributionButtonPosition? attributionButtonPosition;
+  bool useDefaultAttributionPosition = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        const Text("Set attribution position"),
+        Wrap(
+          children: [
+            buildDefaultPositionButton(),
+            buildPositionButton(null),
+            buildPositionButton(AttributionButtonPosition.TopRight),
+            buildPositionButton(AttributionButtonPosition.TopLeft),
+            buildPositionButton(AttributionButtonPosition.BottomRight),
+            buildPositionButton(AttributionButtonPosition.BottomLeft),
+          ],
+        ),
+        Expanded(
+          child: buildMap(
+            attributionButtonPosition,
+            useDefaultAttributionPosition,
+          ),
+        ),
+      ],
+    );
+  }
+
+  ElevatedButton buildDefaultPositionButton() {
+    return ElevatedButton(
+      onPressed: () {
+        setState(() {
+          attributionButtonPosition = null;
+          useDefaultAttributionPosition = true;
+        });
+      },
+      child: const Text("Default"),
+    );
+  }
+
+  ElevatedButton buildPositionButton(AttributionButtonPosition? position) {
+    return ElevatedButton(
+      onPressed: () {
+        setState(() {
+          attributionButtonPosition = position;
+          useDefaultAttributionPosition = false;
+        });
+      },
+      child: Text(position?.name ?? "Null, same as default"),
```

Comment:
```suggestion
      child: Text(position?.name ?? "Null (=platform default"),
```

Reply:
This is a bit weird, but on android the platform default is actually bottomLeft.

So when not specifying any attributionButtonPosition, the flutter default of bottomRight is used. But when setting it to null, the platform default is used, which is bottomRight for web but bottomLeft for android.

Reply:
Should we remove the default set as BottomRight in mapbox-map.dart, to let the native implementations express their own default (bottom left for android, bottom right for web)?

Reply:
I think the way it is right now is actually pretty good: the default on the dart side is bottomRight, so when the developer does not specify anything they get the expected consistent look on all platforms. And if someone explicitly wants the different platform defaults for some reason, they can explicitly set it to null.

Reply:
I would just change the button label as suggested to not confuse developers who look into the example app

Reply:
Done.